docs/pages/server-access/guides/openssh.mdx

```diff
@@ -10,7 +10,12 @@ accept SSH certificates dynamically issued by a Teleport CA.
 
 Using Teleport and OpenSSH has the advantage of getting you up
 and running, but in the long run, we would recommend replacing `sshd` with `teleport`.
-We've outlined these reasons in [OpenSSH vs Teleport SSH for Servers?](https://goteleport.com/blog/openssh-vs-teleport/)
+`teleport` SSH servers have support for multiple features that are incompatible with OpenSSH:
+
+- [Creating ephemeral Unix users when logging into a node](host-user-creation.mdx)
```

Comment:
Technically I think you can do it with plain OpenSSH and PAM so it's not like this is Teleport-exclusive feature.